packaging/linux/common/khiops_env/khiops_env.in

```diff
@@ -64,6 +64,15 @@ get_script_dir() {
 @IS_CONDA_VAR@
 @SET_KHIOPS_DRIVERS_PATH@
 
+if [ -z "$KHIOPS_MPI_VERBOSE" ]; then
+    KHIOPS_MPI_VERBOSE="false"
+fi
+
+if [ ! "$KHIOPS_MPI_VERBOSE" == "true" ]; then
```

Comment:
I would put the negation operator before the opening square bracket, for better legibility.

Reply:
done